### ISSUE TITLE
train NER model using thesaurus and rule annotations

### ISSUE THREAD

**Issue by kdutia** [TheScienceMuseum/heritage-connector-nlp#4]
- [x] generate training data and test data in spaCy format
- [x] run training in 'transfer learning' style
- [x] test performance of new model vs pure NER vs NER w/ thes & dm

**Comment by kdutia** [comment]
see commit [a328d3f](https://github.com/TheScienceMuseum/heritage-connector-nlp/commit/a328d3fadcfa553ee568634b3b5abc77eaa6ee8d)

Continuing training an `en_core_web_lg` instance using labels generated by an `nlp_thes` model (best AAIL results) causes both training losses and test set performance to fluctuate. 

This is likely either because the learning rate is too large or because the labels are too noisy.
